src/frontend/src/actions/profile.js

```diff
@@ -35,6 +35,20 @@ export const sendLoginLink = (email) => (dispatch, getState) => {
   dispatch({ type: "set login message", payload: "SignIn.LoginLinkSent" });
 };
 
+export const setLogout = (authToken) => (dispatch, getState) => {
+  addActionLogItem(`Deleting token cookie`);
+  axios
+    .get("/api/logout/", {
+      headers: {
+        Authorization: `Token ${authToken}`,
```

Comment:
Not redirecting from Django will also fix the issue with the second request to `/` being made with the authorization token header still attached because it's Axios that is following the redirect using the parameters that were initially passed to this `get()` call.

Reply:
Ah ha! Thank you, will implement and update my comment! This is irrelevant now but for my own learning: after trying the delete the header in the response in `views`, I also tried creating an `axios` instance that would be conditionally configured with an authorization token header if `authToken` existed (and the header deleted when not), taking some inspiration from [these lines in treetective](https://github.com/azavea/treetective/blob/develop/client/src/api.ts#L59-L70). This obviously didn't work out so I dropped it, but I initially thought that was because of the state of `authToken`—is it because a redirect will always use the same `axios` instance as the original request so it's not actually creating a "new" instance?

Reply:
> is it because a redirect will always use the same axios instance as the original request so it's not actually creating a "new" instance?

Yup, I think that's exactly what's happening! It's transparently making multiple requests under the hood, but on the "outside" you only see the one Promise with its eventual result.

Reply:
Ah got it, thanks for clarifying!